css/styles.css

```diff
@@ -659,3 +659,286 @@ p#modal-close {
 .modal-btn {
   margin-top: 2rem;
 }
+
+/* Desktop Version */
+
+@media screen and (min-width: 768px) {
+  .headline-section {
+    height: 100vh;
+    background: var(--white) url(../images/background.svg);
+    object-fit: cover;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    border-bottom-left-radius: 75px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-left: 10rem;
+  }
+
+  .nav-menu li a {
+    color: #000;
+    list-style-type: none;
+    font-weight: 450;
+    font-size: 2.1rem;
+  }
+
+  .nav-bar .fa-bars {
+    display: none;
+  }
+
+  ul.navigation-bar.nav-menu {
+    width: 0;
+    height: 0;
+    background: none;
+    display: flex;
+    margin: 0.1rem 60rem;
+    padding-bottom: 3rem;
+    position: fixed;
+  }
```

Comment:
- [ ] Please let's stick to Figma's design as much as possible as requested in the project requirement. Your toolbar section is different compared to Figma's design (see image). Kindly check it.

### Your design
![Screenshot from 2023-03-31 14-41-44](https://user-images.githubusercontent.com/91142432/229159940-ba53d86c-072c-4f39-bb25-16f9b3666adc.png)

### What it supposes to look like 
![Screenshot from 2023-03-31 15-15-20](https://user-images.githubusercontent.com/91142432/229160670-d485b42b-2d41-423b-a13e-b71cf6d730f9.png)